### PR TITLE
[WIP] refactor: use the Core 2.0 API

### PR DESCRIPTION
**Pull request by faustbrian**
Migrates the explorer to the 2.0 API. _Should be finished asap._

### CODE REVIEW

**Comment by dated** [comment]
Needs https://github.com/ArkEcosystem/core/issues/1847

**Comment by faustbrian** [comment]
@dated are any of the things you wanted to have added to Core for 2.1 blocking the completion of this?

**Comment by dated** [comment]
I think everything needed is on the core develop branch, should be good to go.

**Comment by faustbrian** [comment]
Alright, then I will leave this open until 2.1 is on mainnet and then finish this. Can you send a PR to resolve the conflicts?

**Comment by faustbrian** [comment]
@dated could you finish this anytime soon now that 2.1 is on mainnet?

**Comment by ghost** [comment]
@dated The [ci/circleci: test-node-10](https://circleci.com/gh/ArkEcosystem/explorer/669?utm_campaign=vcs-integration-link&amp;utm_medium=referral&amp;utm_source=github-build-link) job is failing as of [a5822cd669d16de092ca585406e3a7e05dfa3407](https://github.com/ArkEcosystem/explorer/commit/a5822cd669d16de092ca585406e3a7e05dfa3407). Please [review the logs for more information](https://circleci.com/gh/ArkEcosystem/explorer/669?utm_campaign=vcs-integration-link&amp;utm_medium=referral&amp;utm_source=github-build-link).

_Once you've pushed the fixes, the build will automatically re-run. Thanks!_

File: resources/views/components/page-headers/wallet/frame.blade.php [hunk 1]
```diff
@@ -13,6 +13,32 @@
...
+                    </span>
+                    @endif
+
+                    @if (!$wallet->isOwnedByExchange())
+                    <span class="">
+                        <x-icon name="app-exchange" size="sm" />
+                    </span>
+                    @endif
+
+                    @if (!$wallet->hasSecondSignature())
```
Comment (faustbrian) [inline resources/views/components/page-headers/wallet/frame.blade.php]: Why `!`?

File: resources/views/components/page-headers/wallet/frame.blade.php [hunk 2]
```diff
@@ -13,6 +13,32 @@
...
+                    </span>
+                    @endif
+
+                    @if (!$wallet->hasMultiSignature())
+                    <span class="">
+                        <x-icon name="app.transactions-multi-signature" size="sm" />
+                    </span>
+                    @endif
+
+                    @if (!$wallet->isOwnedByExchange())
```
Comment (faustbrian) [inline resources/views/components/page-headers/wallet/frame.blade.php]: Why `!`?

File: resources/views/components/page-headers/wallet/frame.blade.php [hunk 3]
```diff
@@ -13,6 +13,32 @@
...
                 @endif
+
+                <div class="grid grid-cols-4 ml-2 divide-x divide-gray-400">
+                    @if (!$wallet->isKnown())
+                    <span class="">
+                        <x-icon name="app-verified" size="sm" />
+                    </span>
+                    @endif
+
+                    @if (!$wallet->hasMultiSignature())
```
Comment (faustbrian) [inline resources/views/components/page-headers/wallet/frame.blade.php]: Why `!`?

File: resources/views/components/page-headers/wallet/frame.blade.php [hunk 4]
```diff
@@ -13,6 +13,32 @@
                         @lang('pages.wallet.address')
                     @endif
                 @endif
+
+                <div class="grid grid-cols-4 ml-2 divide-x divide-gray-400">
+                    @if (!$wallet->isKnown())
```
Comment (faustbrian) [inline resources/views/components/page-headers/wallet/frame.blade.php]: Why `!`?

File: resources/views/components/page-headers/wallet/frame.blade.php [hunk 5]
```diff
@@ -13,6 +13,32 @@
...
+                    <span class="">
```
Comment (faustbrian) [inline resources/views/components/page-headers/wallet/frame.blade.php]: Why an empty class?

File: resources/views/components/page-headers/wallet/frame.blade.php [hunk 6]
```diff
@@ -13,6 +13,32 @@
...
+                        <x-icon name="app-verified" size="sm" />
+                    </span>
+                    @endif
+
+                    @if (!$wallet->hasMultiSignature())
+                    <span class="">
+                        <x-icon name="app.transactions-multi-signature" size="sm" />
+                    </span>
```
Comment (faustbrian) [inline resources/views/components/page-headers/wallet/frame.blade.php]: Why an empty class?

File: resources/views/components/page-headers/wallet/frame.blade.php [hunk 7]
```diff
@@ -13,6 +13,32 @@
...
+                    @endif
+
+                    @if (!$wallet->isOwnedByExchange())
+                    <span class="">
+                        <x-icon name="app-exchange" size="sm" />
+                    </span>
+                    @endif
+
+                    @if (!$wallet->hasSecondSignature())
+                    <span class="">
```
Comment (faustbrian) [inline resources/views/components/page-headers/wallet/frame.blade.php]: Why an empty class?

File: resources/views/components/page-headers/wallet/frame.blade.php [hunk 8]
```diff
@@ -13,6 +13,32 @@
...
+                    @endif
+
+                    @if (!$wallet->hasMultiSignature())
+                    <span class="">
+                        <x-icon name="app.transactions-multi-signature" size="sm" />
+                    </span>
+                    @endif
+
+                    @if (!$wallet->isOwnedByExchange())
+                    <span class="">
```
Comment (faustbrian) [inline resources/views/components/page-headers/wallet/frame.blade.php]: Why an empty class?

**Comment by ItsANameToo** [comment]
rename the class I added as it was just to show you shit

File: resources/lang/en/labels.php [hunk 9]
```diff
@@ -32,5 +32,6 @@
     'verified_address'        => 'Verified Address',
     'votes'                   => 'Votes',
     'wallet_type'             => 'Info',
-
+    'multi_signature'         => 'Multi Signature',
```
Comment (ItsANameToo) [inline resources/lang/en/labels.php]: `Multisignature`

**Pull request merged**